### ISSUE TITLE
redub throws when processing a dub.sdl path dependency

### ISSUE THREAD

**Issue by nordlow** [MrcSnm/redub#33]
In one my packages

referencing a package as 

`dependency "yyjson-d" path="../yyjson-d"`

I'm getting

```
object.Exception@../../.dub/packages/redub/1.17.4/redub/dub_sdl_to_json/source/dub_sdl_to_json.d(87): Whenever dependency has a single attribute, it must be a version. (Parsing yyjson-d)
```

. Can't I use a path dependencies with redub?

I find this exception message difficult to understand and how to fix. Would be great to highlight the offending file locations.

**Comment by MrcSnm** [comment]
This error is on the .sdl conversion, since I've never used SDL, you will may find more issues on that, but I'm working on fixing them. I'll fix that right now and release a new patch.

**Comment by MrcSnm** [comment]
Btw, sorry for those errors, I can only address them quickly right now, most of the conversion were based on inochi-creator since it is one of the biggest .sdl files I found on packages

**Comment by nordlow** [comment]
Thank you!

**Comment by MrcSnm** [comment]
Fixed on v1.17.10

- Get the new version with either `redub update` or simply fetch the new package using dub.

From now on though, I recommend using `redub update` as it will automatically build with redub and with the best version (though I don't know what gonna happen using `dub run`

**Comment by nordlow** [comment]
Works flawlessly now. Awesome responsiveness once again.
